Gemfile

```diff
@@ -34,6 +34,7 @@ gem 'jbuilder', '~> 2.5'
 # gem 'capistrano-rails', group: :development
 
 group :development, :test do
+  gem 'rspec-rails', '3.7.2'
```

Comment:
I highly recommend to add the url link of the gem so that others developer have to hunt down the url to access the documentation for the gem.  So it should be something like this:
gem "rspec-rails", "3.7.2"    # https://github.com/thoughtbot/factory_girl_rails